dashboard/views.py

```diff
@@ -363,10 +373,10 @@ def get_user_assignment_submission(current_user,assignments_in_course_df, course
     return assignment_submissions
 
 
-def hide_score_on_mute(row):
-    if row['grade_muted'] and row['score'] is not None:
-        return None
-    else:return row['score']
+def no_show_avg_score_for_ungraded_assignments(row):
```

Comment:
Could you add some comments to this method? Why are you returning avg_score if the score is None?

Reply:
I have added comments.  ` don't show the avg scores for a student when an individual assignment is not graded as canvas currently don't show it`
I think I return N/A for avg_score when the individual score is not available.